pandas/core/dtypes/dtypes.py

```diff
@@ -1047,14 +1080,32 @@ def __new__(cls, subtype=None):
             )
             raise TypeError(msg)
 
+        if closed is None and subtype is not None:
```

Comment:
if we have this one do we need to warn on L1058? (e.g. which one is getting hit)

Reply:
can you respond to this

Reply:
they are both hit separately